### PR TITLE
Add version command

### PR DESCRIPTION
**Pull request by jeanlaurent**
This add a simple version switch to the CLI, to help determine which version is currently running.

Release is updated to read the version from sources.

### CODE REVIEW

**Comment by jeanlaurent** [comment]
@thaJeztah done in e69fae4
git sha1 would be cool, but require more work, maybe worth an issue ?

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .PHONY: all deps osxkeychain secretservice test validate wincred
 
 TRAVIS_OS_NAME ?= linux
-VERSION = 0.5.1
+VERSION := $(shell grep const credentials/version.go | awk -F'"' '{ print $$2 }')
```
Comment (n4ss) [inline Makefile]: `grep "const Version"` and `print Version $$2`

Comment (n4ss) [inline Makefile]: just in case we have more const variables in this file.

File: Makefile [hunk 2]
```diff
@@ -16,13 +16,14 @@ osxkeychain:
...
 	go build -ldflags -s -o bin/docker-credential-osxkeychain osxkeychain/cmd/main_darwin.go
 
-codesign: osxkeychain
+osxcodesign: osxkeychain
 	$(eval SIGNINGHASH = $(shell security find-identity -v -p codesigning | grep "Developer ID Application: Docker Inc" | cut -d ' ' -f 4))
 	xcrun -log codesign -s $(SIGNINGHASH) --force --verbose bin/docker-credential-osxkeychain
 	xcrun codesign --verify --deep --strict --verbose=2 --display bin/docker-credential-osxkeychain
 
-osxrelease: clean test codesign
+osxrelease: clean vet_osx lint fmt test osxcodesign
```
Comment (n4ss) [inline Makefile]: 👍 good catch, sorry for not checking on my last PR and not making change before..

**Comment by n4ss** [comment]
I prefer the version tag as it is right now personally.

LGTM after the nit "const Version" comment.

File: credentials/version.go [hunk 3]
```diff
@@ -1,4 +1,4 @@
 package credentials
 
-// Version holds a string describing the current version
-const Version = "0.5.1"
+// Version holds a string describing the current versio
```
Comment (thaJeztah) [inline credentials/version.go]: typo: s/versio/version/

File: Makefile (hunk 1 above)
Comment (jeanlaurent) [inline Makefile]: done

File: credentials/version.go (hunk 3 above)
Comment (jeanlaurent) [inline credentials/version.go]: fixed

File: Makefile [hunk 4]
```diff
@@ -1,7 +1,7 @@
 .PHONY: all deps osxkeychain secretservice test validate wincred
 
 TRAVIS_OS_NAME ?= linux
-VERSION = 0.5.1
+VERSION := $(shell grep 'const Version' credentials/version.go | awk -F'"' '{ print $$2 }')
```
Comment (thaJeztah) [inline Makefile]: For a follow-up, perhaps we should consider having a `VERSION` file in the repo

**Pull request merged**